src/main/java/org/alliancegenome/curation_api/jobs/executors/AgmExecutor.java

```diff
@@ -57,17 +58,44 @@ public void runLoad(BulkLoadFile bulkLoadFile) {
 			String speciesName = manual.getDataType().getSpeciesName();
 			String dataType = manual.getDataType().name();
 
-			if (agms == null)
-				agms = new ArrayList<>();
-			
-			bulkLoadFile.setRecordCount(agms.size() + bulkLoadFile.getRecordCount());
-			bulkLoadFileDAO.merge(bulkLoadFile);
+			if(agms != null) {
```

Comment:
This is going to undo the changes put in to fix SCRUM-2696.  We need to create an empty list if `agms` is null so that the load runs with zero entries if a file is submitted without a particular ingest set.  The same applies for all the other executors

Reply:
Why does it need to run without any entries?

Reply:
If you run a DISEASE_ANNOTATION or FULL_INGEST load (only the latter applies to AGMs) then you would expect cleanup to occur for any missing ingest sets.  In SCRUM-2696 we saw that with disease annotations as MGI had been erroneously submitting gene disease annotations - they stopped doing so but the old gene disease annotations were not cleaned up as the  GENE_DiSEASE_ANNOTATION load did not run with an empty set, it was just skipped.

Reply:
But its not the load that needs the empty set... its the cleanup that needs the empty set?